File: FormsPinViewSample/FormsPinViewSample.PCL/ViewModels/MainViewModel.cs
using System;
using System.Windows.Input;
using Xamarin.Forms;

namespace FormsPinViewSample.ViewModels
{
    public class MainViewModel
    {
        public ICommand GithubCommand { get; } = new Command(() => 
        { 
            Device.OpenUri(new Uri("https://github.com/lassana/XamarinFormsPinView"));
        });
    }
}

File: FormsPinViewFSample/FormsPinViewFSample/ViewModels/MainViewModel.fs
namespace FormsPinViewFSample.ViewModels

open System
open System.Windows.Input
open Xamarin.Forms

type MainViewModel() = 
    let githubUri = Uri("https://github.com/lassana/XamarinFormsPinView");

    member this.GithubCommand: ICommand = Command(fun () -> Device.OpenUri(githubUri)) :> ICommand